build.gradle

```diff
@@ -504,11 +504,7 @@ task dockerUpload(type: Exec) {
   dependsOn distDocker
   def imageName = "consensys/quorum-" + repositoryName
```

Comment:
Think we should also add an entry to the change log saying that the consensys/quorum-ethsigner docker repo will be deprecated